### DISCUSSION TITLE
Offline Deletion not synced with server

### DISCUSSION THREAD

**Discussion by FelixKoerner** [immich-app/immich#9612] [Feature Request]
### The bug

While traveling by train, where is not always an internet connection available. This is why it is important for me to be able to move photos or videos to the trash bin while being offline. 
The problem is, that the deletions are not synced with the server. Then I am back online, the deletions are not synced with the server. It is like the media files were never moved to the trash.

### The OS that Immich Server is running on

Debian

### Version of Immich Server

v1.98.2

### Version of Immich Mobile App

v1.98.2

### Platform with the issue

- [X] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
i did not change the example file
```

### Your .env content

```Shell
i did not change the example file
```

### Reproduction steps

```bash
1. Go offline with your mobile
2. Move media files to the trash
3. Go online with your mobile
=> Deletion is ignored
```

**Comment by bradyjoh** [comment]
#23429 is related but not exactly the same. I can't even delete, it just throws an error

**Comment by crisan7** [comment]
i think this is rather a bug than a feature request

if when you press the delete you would receive "delete do not work while offline" then yes it would be a feature request
